### DISCUSSION TITLE
Spam in the forums at digicamcontrol (dot) com

### DISCUSSION THREAD

**Discussion by BStrauss3** [dukus/digiCamControl#413] [General]
Looks like we've had an outbreak of spam posts at the old forums. Probably should clean it up and/or stop new posting.
-----Burton

**Comment by dukus** [comment]
Can you post a link or a screen shot ?

**Comment by BStrauss3** [comment]
https://digicamcontrol.com/phpbb/viewforum.php?f=1


⁣
-----Burton​

**Comment by dukus** [comment]
Thanks, i hope i cleaned up all spam
